.github/workflows/pr-labels.yml

```
@@ -1,70 +1,79 @@
-  checks: write
-  contents: read
   pull-requests: write
   issues: write
 
 jobs:
   update-labels:
     runs-on: ubuntu-latest
     steps:
-      - name: Update PR labels based on reviews
+      - name: Update PR labels
         uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)